src/python/traces/sampling/recipefile.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../../../otel-recipes-schema.json",
+  "id": "python.traces.sampling",
+  "languageId": "python",
+  "signal": "traces",
+  "displayName": "Sampling App",
+  "tags": ["console", "manual"],
```

Comment:
I think we can have a tag for sampling, as it's a good topic. Could you please add it to the schema file? Then you can also add it here `sampling`.

Reply:
Sure. I'll do that.